jmxtrans-output/jmxtrans-output-core/src/main/java/com/googlecode/jmxtrans/model/output/support/opentsdb/OpenTSDBMessageFormatter.java

```diff
@@ -63,19 +64,19 @@
 
 
 	private final boolean mergeTypeNamesTags;
-	private final String hostnameTag;
+	private final boolean hostnameTag;
 
 	public OpenTSDBMessageFormatter(@Nonnull ImmutableList<String> typeNames,
-									@Nonnull ImmutableMap<String, String> tags) throws LifecycleException, UnknownHostException {
-		this(typeNames, tags, DEFAULT_TAG_NAME, null, true, "localhost");
+									@Nonnull ImmutableMap<String, String> tags) throws LifecycleException {
+		this(typeNames, tags, DEFAULT_TAG_NAME, null, true, true);
 	}
 
 	public OpenTSDBMessageFormatter(@Nonnull ImmutableList<String> typeNames,
 									@Nonnull ImmutableMap<String, String> tags,
 									@Nonnull String tagName,
 									@Nullable String metricNamingExpression,
 									boolean mergeTypeNamesTags,
-									@Nullable String hostnameTag) throws UnknownHostException, LifecycleException {
+									Boolean hostnameTag) throws LifecycleException {
```

Comment:
`OpenTSDBMessageFormatter` is an internal component of jmxtrans, where we can ensure that it is never called with `hostnameTag == null`. So I would prefer to have a `boolean` primitive here and not a `Boolean` object.